README.md

```diff
@@ -28,10 +28,20 @@ $> git clone --recursive git@github.com:osclass/Osclass.git
 
 Go to [our site][installing] to get detailed information on installing Osclass.
 
+## Heroku
```

Comment:
In my case, Heroku didn't detect that this repo is PHP app.
Probably because of  `package.json`
So we'd better to set `BUILDPACK_URL=https://github.com/heroku/heroku-buildpack-php`.